### PR TITLE
Destination redshift: also drop cascade on the raw table if necessary

### PR DESCRIPTION
**Pull request by edgao**
this only affects users who create non-late-binding views on top of the raw tables, which is presumably a very narrow subset. The final table already respects the dropCascade option.

Added a simple integration test to RedshiftStorageOperations to exercise this option.

### CODE REVIEW

File: airbyte-integrations/connectors/destination-redshift/src/test-integration/kotlin/io/airbyte/integrations/destination/redshift/RedshiftS3StagingStorageOperationTest.kt [hunk 1]
```diff
@@ -197,6 +234,22 @@ class RedshiftS3StagingStorageOperationTest {
         }
     }
 
+    private fun getRedshiftObjects(dropCascade: Boolean): Pair<RedshiftStagingStorageOperation, RedshiftDestinationHandler> {
```
Comment (edgao) [inline airbyte-integrations/connectors/destination-redshift/src/test-integration/kotlin/io/airbyte/integrations/destination/redshift/RedshiftS3StagingStorageOperationTest.kt:237]: this method name sucks, please suggest something better 😓

Comment (stephane-airbyte) [reply airbyte-integrations/connectors/destination-redshift/src/test-integration/kotlin/io/airbyte/integrations/destination/redshift/RedshiftS3StagingStorageOperationTest.kt:237]: the split of different objects sucks. It is what it is

Comment (stephane-airbyte) [reply airbyte-integrations/connectors/destination-redshift/src/test-integration/kotlin/io/airbyte/integrations/destination/redshift/RedshiftS3StagingStorageOperationTest.kt:237]: actually, a simple one would be to expose the stagingOperation.handler, so you can return only the storageOperation

**Review by stephane-airbyte — APPROVED** [review]
LGTM. A very simple suggestion on changing `getRedshiftObjects`

File: airbyte-integrations/connectors/destination-redshift/src/test-integration/kotlin/io/airbyte/integrations/destination/redshift/RedshiftS3StagingStorageOperationTest.kt (hunk 1 above)
Comment (edgao) [reply airbyte-integrations/connectors/destination-redshift/src/test-integration/kotlin/io/airbyte/integrations/destination/redshift/RedshiftS3StagingStorageOperationTest.kt:237]: derp. I forgot that we already have the underlying jdbcDatabase object >.> so we only need to return the storage op object anyway

**Pull request merged**